lib/Nodes/DefinitionListNode.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Nodes;
+
+use Doctrine\RST\Parser\DefinitionList;
+
+abstract class DefinitionListNode extends Node
```

Comment:
This abstract class only serves as a way to share a constructor: can we get rid of it?

Reply:
So get rid of the base abstract class and duplicate the constructor in the different implementations of this node type? I only have the HTML one right now but I need to write the LaTex one still.

Reply:
Yes: inheritance is not a code reuse mechanism, so the duplication is OK, and will actually ease migration of further use-case scenarios

Reply:
The `NodeFactory::createDefinitionList` method returns the base abstract class. The way it works currently is you have the base node implementation, and then you extend for HTML, LaTex, etc. This could be better but refactoring that would probably be out of scope for this PR. I can duplicate the constructor but leave the base node for now.

Reply:
Long term I think we need a way to represent a node that doesn't care or know if it is being rendered with HTML, LaTex, etc. and then have a renderer interface that we implement for html, latex, etc.

Reply:
Issue logged here for that https://github.com/doctrine/rst-parser/issues/36